conf/machine/include/imx-base.inc

```diff
@@ -637,7 +637,7 @@ WKS_FILE_DEPENDS ?= " \
     virtual/bootloader \
     \
     e2fsprogs-native \
-    bmap-tools-native \
+    bmaptool-native \
```

Comment:
This is only compatible with scarthgap.

Reply:
Correct. I did not see `LAYERSERIES_COMPAT` was set to more branches than `scarthgap` or `master`. Maybe we should create a `nanbield` branch not including this patch?

Reply:
@thochstein submitted a patch to OE-Core that added fields for backward compatibility, allowing for reversion.